.github/workflows/README.md

```
@@ -0,0 +1,47 @@
+
+## General
+
+Any changes to files in this directory are flagged when pull requests are run. Make changes only on the advice of a contributor.
+
+## YAML Workflow Files
+
+The YAML files in this directory have very specific roles depending on the type of workflow.
+
+Whenever possible you must ensure that:
+1. The file roles below are maintained
+1. The sequence of the jobs in the workflows are maintained using [GitHub Action dependencies](https://docs.github.com/en/actions/learn-github-actions/managing-complex-workflows). 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)